scripts/src/adaptTranslationCsv.ts

```diff
@@ -0,0 +1,70 @@
+import csvParse = require("csv-parse/lib/sync");
+import { readFileSync, writeFile } from "fs";
+import { path } from "./utils";
+import { CsvParse, TranslationData } from "./admin/translations";
+import { ExportToCsv, Options as ExportCsvOptions } from "export-to-csv";
+
+function getOfferTranslation(inputCsv: TranslationData) {
+	return inputCsv.reduce((dict, [type, key, defaultStr, translation]) => {
+		if (!translation || !translation.length) {
+			return dict;
+		}
+		dict[defaultStr] = translation;
+		return dict;
+	}, {} as { [defaultStr: string]: string });
+
+}
+
+async function addTranslationTo(csv: TranslationData, fromDict: { [defaultStr: string]: string }) {
+	return csv.map(([type, key, defaultStr, __, charLimit]) => {
+		return {
+			Type: type,
+			Key: key,
+			Default: defaultStr,
+			Translation: fromDict[defaultStr] || defaultStr,
+			"Character Limit": charLimit,
+		};
+	});
+}
+
+function writeCsvDataToFile(data: any[], fileName: string) {
+	const options: ExportCsvOptions = {
```

Comment:
is the type annotation here necessary? 
the compiler should be ok with it even if you don't explicitly tell it that options is of type `ExportCsvOptions`, as it use duck typing it should be fine